### PR TITLE
Basic Frontend using Svelte

### LINKED ISSUE #134: Frontend in `Svelte`

### PR DESCRIPTION
**Pull request by rohan-b-84**
Closes #134 

Here is the basic frontend for GYFT made using svelte :)

Desktop View
Mobile View

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# create-svelte
```
Comment (chirag-ghosh) [inline frontend/README.md:1]: Let's replace this with a contextualised readme.

File: frontend/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1769 @@
+{
```
Comment (chirag-ghosh) [inline frontend/package-lock.json:1]: I guess you are using npm. Wanna try yarn or pnpm? [Why?](https://dev.to/ayoub3bidi/lets-settle-things-out-2-npm-vs-yarn-vs-pnpm-5e04#:~:text=For%20NPM%20It%20is%20still,PNPM%20is%20faster%20than%20Yarn.)

File: frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+	"name": "frontend",
```
Comment (chirag-ghosh) [inline frontend/package.json:2]: Replace with gyft

File: frontend/src/app.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (chirag-ghosh) [inline frontend/src/app.html:1]: Let's add metatags here.

File: frontend/src/components/Form.svelte [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+      <div>
+        <h3>ERP Roll Number :</h3>
+        <input
+          bind:value={formData.rollNo}
+          placeholder="Enter Your ERP Roll Number"
+        />
+      </div>
+      <button
+        on:click={() => {
+          console.log(formData);
```
Comment (chirag-ghosh) [inline frontend/src/components/Form.svelte:25]: Remove

**Review by chirag-ghosh — CHANGES_REQUESTED** [review]
Let's also add the functionality in this PR itself.

Also, use scss instead of css.

**Comment by rohan-b-84** [comment]
not needed anymore as made one using react

**Pull request closed without merging**